helm/aladdin-demo/templates/aladdin-demo.deploy.yaml

```diff
@@ -0,0 +1,69 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: {{ .Chart.Name }}
+  labels:
+    project: {{ .Chart.Name }}
+    name: {{ .Chart.Name }}
+    app: {{ .Chart.Name }}
+    githash: {{.Values.deploy.imageTag}}
+spec:
+  replicas: {{.Values.deploy.replicas}}
+  strategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        project: {{ .Chart.Name }}
+        name: {{ .Chart.Name }}
+        app: {{ .Chart.Name }}
+        githash: {{.Values.deploy.imageTag}}
+    spec:
+      # Number of seconds for the containers to perform a graceful shutdown,
+      # after which it is voilently terminated. This defaults to 30s, most apps may not need to change it
+      terminationGracePeriodSeconds: {{ .Values.deploy.terminationGracePeriod }}
+      containers:
+############# aladdin-demo app container #################
+      # falcon app container that runs with uwsgi server
+      - name: {{ .Chart.Name }}
+        # Docker image for this container
+        image: {{ .Chart.Name }}:{{ .Values.deploy.imageTag }}
```

Comment:
Still need {{ .Values.deploy.ecr }} - this is set to empty on local so it doesn't break minikube. However, once you publish and deploy to KDEV, this needs to be there to find the image.

Reply:
Makes sense, would we also need `imagePullSecrets: - name: aws`? This is a FiveStars specific thing right? Is this appropriate to have it in this demo?

Reply:
I'm actually not sure, some projects were working without it, but it doesn't hurt to put this in.